CHANGELOG.md

```diff
@@ -266,15 +266,6 @@ kind of thing rather than trying to weave it into the main module logic itself.
 - Fix fedora support by using systemd.
 - Initdb should create xlogdir if set.
 
-##2014-08-27 - Supported Release 3.4.3
-###Summary
-
-This release fixes Ubuntu 10.04 with Facter 2.2.
```

Comment:
this is for a really old release, the fix has been in for a long time either way.